modules/cactus-web/src/Breadcrumb/Breadcrumb.tsx

```diff
@@ -83,6 +86,26 @@ const BreadcrumbBase = (props: BreadcrumbProps): React.ReactElement => {
     [toggle, setFocus]
   )
 
+  const checkEllipsisVersion = React.useCallback(() => {
+    setEllipsisVersion(false)
+    const parentWidth = mainNavContainer.current?.parentElement?.getBoundingClientRect().width
+    const pivotWidth = pivotBreadcrumb.current?.getBoundingClientRect().width
+    if ((pivotWidth && parentWidth && pivotWidth >= parentWidth) || (isTiny && childrenCount > 2)) {
+      setEllipsisVersion(true)
+    }
+  }, [childrenCount, isTiny])
+
+  React.useLayoutEffect(() => {
+    checkEllipsisVersion()
+    const handleResize = () => {
+      checkEllipsisVersion()
+    }
+    window.addEventListener('resize', handleResize)
+    return () => {
+      window.removeEventListener('resize', handleResize)
+    }
+  }, [checkEllipsisVersion])
```

Comment:
It would be better to implement the `checkEllipsisVersion` function in here, since this is where it's being used:
```suggestion
  React.useLayoutEffect(() => {
    const checkEllipsisVersion = () => {
      // ...etc
    }
    checkEllipsisVersion()
    window.addEventListener('resize', checkEllipsisVersion)
    return () => {
      window.removeEventListener('resize', checkEllipsisVersion)
    }
  }, [childrenCount, isTiny])
```